daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/LargeTransactionTest.scala

```diff
@@ -101,7 +101,8 @@ class LargeTransactionTest extends AnyWordSpec with Matchers with BazelRunfiles
   private def report(name: String, quantity: Quantity[Double]): Unit =
     println(s"$name: $quantity")
 
-  private val engine: Engine = Engine.DevEngine()
+  private val engine: Engine =
+    new Engine(new EngineConfig(LanguageVersion.DevVersions, valueNormalization = false))
```

Comment:
Because the test inspects record values embedded in a transaction, and it looks up fields by name
I expect it would be possible to change the test to avoid use of the field name, and so work with normalized values.

Reply:
But I think I prefer to leave the test working with type-annotated values.